bzkanban.js

```diff
@@ -86,6 +88,7 @@ function initNav() {
 
     nav.appendChild(createQueryFields());
     if (bzAllowEditBugs) {
+        nav.appendChild(createBacklogButton());
         nav.appendChild(createBacklogTarget());
```

Comment:
I'm also thinking it'd be better if we put the show backlog button in the `actions` span on the right hand side.